openassessment/xblock/staff_area_mixin.py

```diff
@@ -492,9 +492,36 @@ def reschedule_unfinished_tasks(self, data, suffix=''):  # pylint: disable=W0613
                 'msg': self._(u"An error occurred while rescheduling tasks: {}".format(ex))
             }
 
+    def clear_submission(self, user_id, course_id, item_id):
+        """
+            This method is called to reset student state for a given problem. It will
```

Comment:
Copied from `cancel_submission`, which does the same thing. I'll clean both of them up.